cfep-23.md

```diff
@@ -0,0 +1,75 @@
+
+However, it suffers from a few issues that limit its usability within our community. The main concerns are:
+
+- End-to-end encryption fails sometimes, leaving the users unable to read previous messages, or have their messaged read by others. Synchronization of keys across clients is necessary to fix it, but in some other cases the issue has dissapeared spontaneously.
+- Some messages are considered unread for months at a time, and there's no way to find them and mark them as read.
+- The thread system offers very limited ways of tracking conversations and making them discoverable. It's very difficult to follow on topics started hours ago, let alone days or weeks ago.
+- Scoped discussions require a lot of discipline from the users, since threads are optional, and the real-time chat is preferred in the UI.
+
+Zulip, on the other side, is a free to use, open source, real-time discussion board. It has web, desktop and mobile clients. Each community has its Zulip instance, where several "channels" can be created. Each channel hosts a number of threads or topics. Instant messaging is also available within the Zulip instance. Users can also create a new account or log in with their Github credentials, among others.
+
+Zulip offers some features we would consider beneficial to the conda-forge community:
+
+- Threads (or topics) are encouraged and the default user experience.
+- Each topic must have a title that promotes well-scoped discussions, which can be marked as solved when the issue has been addressed.
+- Moderation tools are available to split threads into new ones in case the conversation goes off-topic.
+- Users can "follow" threads they find interesting, which has its own dedicated view. This makes it easier to track updates to conversations you have started or participated in.
+- Private rooms are also available, but they don't offer end-to-end encryption. For this matter, a sensitive communications protocol is proposed below.
+
+## Proposal
+
+After accepting this proposal, we will implement the following decisions:
+
+- The Zulip instance published at `conda-forge.zulipchat.com` will be open to the public with world-readable permissions.
+- The General channel will be the replacement for the `conda-forge/conda-forge.github.io` chat room. This will be reflected in the conda-forge.org documentation, and reminded periodically in the Element room.
+- The conda-forge core team will move its private communications to the Zulip instance, and stop using the Element rooms. A last message will be published in the Element core chat room with the details on how to transition to the Zulip instance. 
+- New channels will be created in the instance as considered necessary (e.g. infrastructure, compilers, etc).
+- The Zulip instance will be treated as a public message board. No private chat or private rooms for folks besides admins or owners will be enabled.
```

Comment:
Shall we include "moderators" here? Thinking of non-core CoC committee members (in the future), staged-recipes, etc.

Reply:
I did not on purpose, but I agree. We can add moderators.